### PR TITLE
Fix code-expansion arc-green theme bug

### LINKED ISSUE #10179: [Theme] [Bug] arc-green
gitea version: current master

code-expansion area has wrong color:
Bildschirmfoto zu 2020-02-07 18-07-18

### PR DESCRIPTION
**Pull request by 6543**
close #10179

new look:
Bildschirmfoto zu 2020-02-07 19-16-06

### CODE REVIEW

File: web_src/less/themes/theme-arc-green.less [hunk 1]
```diff
@@ -1109,6 +1109,9 @@ input {
 .lines-code .hljs {
     background-color: #2a2e3a !important;
 }
+.blob-excerpt {
```
Comment (jolheiser) [inline web_src/less/themes/theme-arc-green.less]: To override the `gitea` theme
```suggestion
td.blob-excerpt {
```

File: web_src/less/themes/theme-arc-green.less [hunk 2]
```diff
@@ -1109,6 +1109,9 @@ input {
...
+    background-color: #252934 !important;
```
Comment (jolheiser) [inline web_src/less/themes/theme-arc-green.less]: In combination with the other suggestion, `!important` is no longer required as this loads later and therefore takes precedence.
```suggestion
    background-color: #252934;
```
-----
Alternatively, up to other opinions as well but the color could mirror what the regular theme does by making the excerpt match the line number side-bar. See #10181 for a screenshot. Yours is more noticeable, though, so I have no strong opinion either way.
```suggestion
    background-color: #2e323e;
```

**Review by jolheiser — CHANGES_REQUESTED** [review]
At the very least I think we should avoid `!important` if we can, but after that I will approve regardless of other suggestion.

File: web_src/less/themes/theme-arc-green.less (hunk 2 above)
Comment (6543) [reply web_src/less/themes/theme-arc-green.less]: it was intended to make it noticable

**Comment by silverwind** [comment]
Would suggest `background-color: rgba(0,0,0,0.1)` instead of defining a new RGB color just for this. Also, the color difference seems really low on your screenshot, I would go with something between 0.1 and 0.2.

File: web_src/less/themes/theme-arc-green.less (hunk 1 above)
Comment (6543) [reply web_src/less/themes/theme-arc-green.less]: it was als intended to leave td because it is more flexible when template changes ocour

Comment (6543) [reply web_src/less/themes/theme-arc-green.less]: should I leave it or add the `td` ? but then it should also be added to `.lines-code`

Comment (jolheiser) [reply web_src/less/themes/theme-arc-green.less]: If you take a look at #10181 it only needs to be added to this one.   
I would prefer to avoid an `!important` if we can.

**Comment by 6543** [comment]
@silverwind your solution doesn't work ...

File: web_src/less/themes/theme-arc-green.less [hunk 3]
```diff
@@ -1107,7 +1107,10 @@ input {
 .lines-code ol,
 .lines-num .hljs,
 .lines-code .hljs {
-    background-color: #2a2e3a !important;
+    background-color: #2a2e3a;
```
Comment (jolheiser) [inline web_src/less/themes/theme-arc-green.less]: This change seems unrelated. This CSS doesn't apply to diffs as far as I know (at least I didn't see it in testing).

**Comment by silverwind** [comment]
It work does for me:




```` css
td.blob-excerpt {
    background-color: rgba(0,0,0,.2);
}
````

**Comment by 6543** [comment]
just noticed it need `td` if we like to remove important ...

**Comment by 6543** [comment]
@silverwind I'll test your solutin with td again ...

**Comment by silverwind** [comment]
Yes, match the selector exactly from the base style.

**Comment by 6543** [comment]
ok works - changed
PS: I used 0.1 because 0.2 was to intensive ... in my option ...

**Comment by silverwind** [comment]
0.1 seems to light to me. Maybe meet in the middle at 0.15?

**Comment by jolheiser** [comment]
And when it's settled, a new screenshot please. 🙂

**Comment by 6543** [comment]
@jolheiser pull desc updated

**Review by silverwind — APPROVED** [review]

**Review by jolheiser — APPROVED** [review]

**Review by guillep2k — APPROVED** [review]

**Review by sapk — APPROVED** [review]
